### DISCUSSION TITLE
justpy behind proxy

### DISCUSSION THREAD

**Discussion by WolfgangFahl** [elimintz/justpy#402] [Q&A]
I am trying to use justpy behind an apache reverse proxy and run into the same problem as:
https://stackoverflow.com/q/68866920/1497139

Im using an Apache 2.4 reverse proxy  configuration with the following configuration snippet:
```apachectl
 ProxyPreserveHost On
  ProxyRequests Off
  ProxyPass / http://localhost:8080/
  ProxyPassReverse / http://localhost:8080/
```

The justpy webpage appears via the proxy but the "reload" button appears. 
**How can this be avoide what would be the configuration workaround?**

**Comment by WolfgangFahl** [comment]
I assume https://www.serverlab.ca/tutorials/linux/web-servers-linux/how-to-reverse-proxy-websockets-with-apache-2-4/ might habe some info

**Comment by WolfgangFahl** [comment]
![grafik](https://user-images.githubusercontent.com/1336221/181994829-2512161c-5696-4d7c-8c20-28a0f5235687.png)

**Comment by WolfgangFahl** [comment]
tried along https://stackoverflow.com/a/70087554/1497139 to no avail

**Comment by WolfgangFahl** [comment]
https://stackoverflow.com/a/35141086/1497139 got me a lot more mileage and seems to at least mostly work.

**Comment by falkoschindler** [comment]
This discussion seems related: https://github.com/elimintz/justpy/pull/258

We ended up forking JustPy and changing paths in the template directory (among a few other things): https://github.com/elimintz/justpy/compare/master...zauberzeug:justpy:master

**Reply by platinops** [reply]
You could also have a look at #208 for some insight on how to use justpy with multiple workers behind nginx.

**Comment by WolfgangFahl** [comment]
see also https://github.com/encode/starlette/issues/604
